src/Symfony/Bundle/SecurityBundle/Tests/Functional/SecurityTest.php

```diff
@@ -108,10 +108,8 @@ public static function userWillBeMarkedAsChangedIfRolesHasChangedProvider(): arr
         ];
     }
 
-    /**
-     * @testWith    ["form_login"]
-     *              ["Symfony\\Bundle\\SecurityBundle\\Tests\\Functional\\Bundle\\AuthenticatorBundle\\ApiAuthenticator"]
-     */
+    #[TestWith(['form_login'])]
+    #[TestWith(['Symfony\Bundle\SecurityBundle\Tests\Functional\Bundle\AuthenticatorBundle\ApiAuthenticator'])]
```

Comment:
What about using `::class` here:


```suggestion
    #[TestWith([ApiAuthenticator::class])]
```